src/common/signal/signal_action.h

```diff
@@ -71,6 +71,20 @@ class SignalAction : public NotCopyable {
    * Additionally, two guard pages will be allocated to bookend the usable area.
    */
   const size_t altstack_size_;
+  /**
+   * Determine the number of bytes to allocate to altstack_size_.
+   * Makes sure that it is a multiple of the system page size.
+   */
+  size_t DetermineAltStackSize() const {
+    // The size of altstack_ should be at least 4 * page size or size MINSIGSTKSZ if it is
+    // greater. This size needs to be a multiple of the system page size.
+    const size_t page_size = px::system::Config::GetInstance().PageSizeBytes();
```

Comment:
`guard_size_` is set to the page size. Can we reuse that rather than creating another variable for referencing page size?